lib/helpers/configuration.rb

```diff
@@ -20,10 +20,11 @@ def configuration
     yield self
   end
 
-  def define_setting(name, default = nil)
+  def define_setting(name, default = nil, allow_nil: true)
     class_variable_set("@@#{name}", default)
 
     define_class_method "#{name}=" do |value|
+      raise "#{name} cannot be nil." if value.nil? && !allow_nil
```

Comment:
Good point! :smile: 

Should be `share_meow_client.rb` and ShareMeowClient initializer.